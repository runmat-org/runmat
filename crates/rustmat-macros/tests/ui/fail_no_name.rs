use rustmat_macros::matlab_fn;

#[matlab_fn]
fn foo() {}

fn main() {}
