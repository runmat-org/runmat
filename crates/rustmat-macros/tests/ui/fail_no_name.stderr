error: custom attribute panicked
 --> tests/ui/fail_no_name.rs:3:1
  |
3 | #[matlab_fn]
  | ^^^^^^^^^^^^
  |
  = help: message: expected `name = "..."` argument
